packages/react/src/stories/ic-hero.stories.mdx

```diff
@@ -55,6 +55,63 @@ export const ControlledReset = () => {
   );
 };
 
+### Playground
+
+export const defaultArgs = {
+  aligned: "left",
+  backgroundImage: image,
+  contentAligned: "left",
+  disableBackgroundParallax: false,
+  heading: "Hero heading",
+  secondaryHeading: "Secondary Heading",
+  secondarySubheading: "This is a secondary description.",
+  size: "default",
+  subheading: "Hero description. This is a Hero component, it should be used as a page heading.",
+};
+
+<Canvas>
+  <Story
+    args={defaultArgs}
+    name="Playground"
+    parameters={{ layout: "fullscreen" }}
+    argTypes={{
```

Comment:
could we make the background image be a select (with an empty value in the list too ) rather than a string? that way you could switch between the image & none

Reply:
I could do, but I thought it'd be more flexible if I left it as a string, as it defaults to an image we have, but then it also can be changed to the URL or any other image. So the string allows the flexibility of the current image, no image, and any other image. Whereas a select wouldn't allow any other image. Let me know if you still think we go down the route of using a switch though